contracts/utils/Arrays.sol

```diff
@@ -57,7 +57,7 @@ library Arrays {
     function unsafeAccess(address[] storage arr, uint256 pos) internal pure returns (StorageSlot.AddressSlot storage) {
         bytes32 slot;
         // We use assembly to calculate the storage slot of the element at index `pos` of the dynamic array `arr`
-        // following https://docs.soliditylang.org/en/v0.8.17/internals/layout_in_storage.html#mappings-and-dynamic-arrays.
+        // following https://docs.soliditylang.org/en/latest/internals/layout_in_storage.html#mappings-and-dynamic-arrays.
```

Comment:
I think we don't use latest here in case the docs ever change in a way that would make the link invalid. For example the tag could change. I'd be ok with changing that to `v0.8.20`.

Reply:
Yeah I prefer using a fixed version.

Reply:
We have a few others using `latest`. I think we had this conversation in another PR and we decided to do `latest` iirc. No problem for now, just flagging in case it's worth discussing a better strategy.

Reply:
I would vote for using a fixed version in all links to Solidity docs.

Reply:
I'd vote for consistency, and I'd push for using `0.8.x` or something that doesn't fixes the docs to an specific patch.